spec.html

```diff
@@ -46398,8 +46399,8 @@ <h1>The %AsyncIteratorPrototype% Object</h1>
         <p>All objects defined in this specification that implement the AsyncIterator interface also inherit from %AsyncIteratorPrototype%. ECMAScript code may also define objects that inherit from %AsyncIteratorPrototype%. The %AsyncIteratorPrototype% object provides a place where additional methods that are applicable to all async iterator objects may be added.</p>
       </emu-note>
 
-      <emu-clause id="sec-asynciteratorprototype-asynciterator">
-        <h1>%AsyncIteratorPrototype% [ @@asyncIterator ] ( )</h1>
+      <emu-clause oldids="sec-asynciteratorprototype-asynciterator" id="sec-asynciteratorprototype-%symbol.asynciterator%">
```

Comment:
```suggestion
      <emu-clause oldids="sec-asynciteratorprototype-asynciterator" id="sec-%asynciteratorprototype%-%symbol.asynciterator%">
```
because we usually include the base object's percents in the id. (The status quo id is anomalous on this point.)

Reply:
Either one WFM. I'm not too worried about consistency in clause IDs.